apps/settings/js/panels/call/call_settings_task_scheduler.js

```diff
@@ -0,0 +1,35 @@
+define(function(require) {
+  'use strict';
+
+  var CallSettingsTaskSchedulerConstant =
+    require('panels/call/call_settings_task_scheduler_constant');
+  var Module = require('modules/base/module');
+  var TaskScheduler = require('modules/base/task_scheduler');
+
+  var CallSettingsTaskScheduler = Module.create(function() {
+    this._taskScheduler = TaskScheduler();
+  });
+
+  CallSettingsTaskScheduler.prototype.TASK_TYPES =
+    CallSettingsTaskSchedulerConstant.TASK_TYPES;
+
+  CallSettingsTaskScheduler.prototype.observe = function() {
+    this._taskScheduler.observe.apply(this._taskScheduler, arguments);
+  };
+
+  CallSettingsTaskScheduler.prototype.enqueue = function(type, func) {
```

Comment:
I would suggest to make the function signature the same as `TaskScheduler.enqueue`.